### DISCUSSION TITLE
Move download of vpn contrib to /config to save download time

### DISCUSSION THREAD

**Discussion by ksurl** [haugene/docker-transmission-openvpn#2591] [Ideas]
seems like the longest part of container startup is waiting for the download and unzipping it. can it be moved to save to /config and download via git so you only get the diffs on startup? I can give it a shot at a PR if this is something that the team would be interested in.

**Comment by pkishino** [comment]
You mean build a cache? Yeah sure, feel free to submit a PR

**Reply by ksurl** [reply]
I just read the fetch-external-configs.sh file and didn't realize it already had a git clone option.
